src/dist/replication/lib/replica_stub.cpp

```diff
@@ -598,30 +597,16 @@ void replica_stub::initialize(const replication_options &opts, bool clear /* = f
                smax);
 
         if (err == ERR_OK) {
-            if (smax != pmax) {
```

Comment:
I guess smax is the max decree of share log, and pmax is the max decredd of private log. Is that right?

Reply:
Yeah